domino/core/pom.xml

```diff
@@ -20,6 +20,12 @@
 				<extensions>true</extensions>
 				<configuration>
 					<singleton>true</singleton>
+					<manifestLocation>META-INF</manifestLocation>
```

Comment:
don't know. The /target will be deleted on "mvn clean", so this manifest is never deleted. As it is in .gitignore, it should be no problem